plugins/modules/ovirt_disk.py

```diff
@@ -65,7 +65,7 @@
                if you want to upload the disk even if the disk with C(id) or C(name) exists,
                then please use C(force) I(true). If you will use C(force) I(false), which
                is default, then the disk image won't be uploaded."
-            - "Note that to upload iso the C(format) should be 'raw'"
+            - "Note that in order to upload iso the C(format) should be 'raw'".
```

Comment:
I think `.` should be moved ahead of `"`

Reply:
You are right, my bad :)
Fixed.